packages/components/psammead-grid/src/index.jsx

```diff
@@ -287,13 +287,15 @@ const Grid = ({
 }) => {
   const renderChildren = () =>
     React.Children.map(children, child => {
-      const isNestedGridComponent = child.type === Grid;
+      if (child) {
+        const isNestedGridComponent = child.type === Grid;
 
-      if (isNestedGridComponent) {
-        return React.cloneElement(child, {
-          parentColumns: otherProps.columns,
-          parentEnableGelGutters: otherProps.enableGelGutters,
-        });
+        if (isNestedGridComponent) {
+          return React.cloneElement(child, {
+            parentColumns: otherProps.columns,
+            parentEnableGelGutters: otherProps.enableGelGutters,
+          });
+        }
       }
       return child;
```

Comment:
I think it's possible to remove this line because `undefined` will be returned by default which also shouldn't render anything.

Reply:
I believe we need this so that the `renderChildren` function returns a child when child is defined.

Reply:
Yeh you're right ignore me!